ethereum/docs/src/run/operate_bridge.md

```diff
@@ -0,0 +1,179 @@
+
+The response bodies are more straight forward:
+
+**Proof generation**
+
+| Name | Byte offset | Description |
+|------|-------------|-------------|
+| Proof type  | 0           | Type of the proof that the payload contains. `0` for STARK and `1` for SNARK                                                                                                             |
+| Proof | 1           | Bytes representing a JSON serialized [`SphinxProofWithPublicValues`](https://github.com/argumentcomputer/sphinx/blob/36f3f9072dc187612640e2725a2f7524cf2f2215/sdk/src/proof.rs#L21-L28). |
+
+**Proof verification**
+
+| Name                          | Byte offset | Description                                                                                |
+|-------------------------------|-------------|--------------------------------------------------------------------------------------------|
+| Successful proof verification | 0           | A `0` (fail) or `1` (success) byte value representing the success of a proof verification. |
+
+## Adapt the verifier
+
+In the following section we will touch upon how a verifier contract has to be updated
+depending on a use case. However, it has to be kept in mind that some core
+data will have to be passed even thought some modifications have to be done 
+for different use cases.
+
+### Core data
+
+> **Note**
+> 
+> The following documentation will be for SNARK proofs, as they are the only 
+> proofs that can be verified on our home chains.
+
+The core data to be passed to any verification contrtact are the following:
+- Verifying key: A unique key represented as 32 bytes, related to the program that is meant to be verified
+- Public values: Serialized public values of the proof
+- Proof: The serialized proof to be verified
+
+**Verifying key**
+
+The verifying key for a program at a given commit can be found in its fixture file
+in the format of a hexified string prefixed by `0x`. There is [one file for the committee
+change](https://github.com/argumentcomputer/zk-light-clients/blob/dev/ethereum/pact/fixtures/epoch_change_fixture.json) 
+program and one file for [the inclusion program](https://github.com/argumentcomputer/zk-light-clients/blob/dev/ethereum/pact/fixtures/inclusion_fixture.json).
+
+**Public values**
+
+The public values and serialized proof data can be found through the type [`SphinxProofWithPublicValues`](https://github.com/argumentcomputer/sphinx/blob/36f3f9072dc187612640e2725a2f7524cf2f2215/sdk/src/proof.rs#L21-L28)
+returned as an HTTP response body by the prover. 
+
+The public values can be found under the `public_values` property and are already
+represented as a `Buffer` which data are to be transmitted to the verifier contract.
+In the fixture files we leverage in our codebase, the public values are represented
+as a hexified string prefixed by `0x`.
+
+**Proof**
+
+The proof data to be passed to the verifier contract will depend on the chain
```

Comment:
We may explain the reason for this difference at a high level: Kadena runs the verifier in the form of node code, that is the node has special features for verifying such proofs. Aptos runs the verifier in a smart contract. We expect the difference in proof formats to perdure according to whether the destination verifier runs in a SC / natively.